plugins/jobs/src/js/data/JobModel.ts

```diff
@@ -289,5 +382,9 @@ export const resolvers = (): IResolvers => ({
 
 export default makeExecutableSchema({
   typeDefs,
-  resolvers: resolvers()
+  resolvers: resolvers({
+    fetchJobs: MetronomeClient.fetchJobs,
+    fetchJobDetail: MetronomeClient.fetchJobDetail,
+    pollingInterval: 2000
```

Comment:
What is the constant that goes here again?

Reply:
`Config.getRefreshRate()`

Reply:
Thanks.